doc/api/repl.md

```diff
@@ -382,6 +382,8 @@ added: v0.1.91
      `undefined`. Defaults to `false`.
   * `writer` {Function} The function to invoke to format the output of each
      command before writing to `output`. Defaults to [`util.inspect()`][].
+  * `completer` {Function} An optional function that will used for custom Tab
```

Comment:
"will _be_ used"
Nit: I would prefer an active tense usage here, such as `An optional function that is used for custom tab auto completion`.